### ISSUE TITLE
bug: @ionic/react unexpected warning "No IonPage was found to render"

### ISSUE THREAD

**Issue by kodemi** [ionic-team/ionic#19977]
# Bug Report

**Ionic version:**
[x] **4.11.x**

**Current behavior:**
Ionic throws warning "No IonPage was found to render. Make sure you wrap your page with an IonPage component."

**Expected behavior:**
No warning

**Steps to reproduce:**
1. Go to https://codesandbox.io/s/jolly-leaf-lddk1
2. Navigate in Browser pane to https://lddk1.csb.app/page1
3. Click button "To Root"
4. Page 2 opened (https://lddk1.csb.app/page2). In console pane you could see the warning.

**Related code:**

```javascript
const App = () => (
  <IonApp>
    <IonReactRouter>
      <IonRouterOutlet>
        <Route exact path="/page1" component={Page1} />
        <Route exact path="/page2" component={Page2} />
        <Redirect exact from="/" to="/page2" />
      </IonRouterOutlet>
    </IonReactRouter>
  </IonApp>
);

const Page1 = ({ history }) => (
  <IonPage>
    <IonContent>
      <h1>Page 1</h1>
      <IonButton onClick={() => history.replace("/")}>To Root</IonButton>
    </IonContent>
  </IonPage>
);

const Page2 = () => (
  <IonPage>
    <IonContent>
      <h1>Page 2</h1>
    </IonContent>
  </IonPage>
);
```

**Comment by OoDeLally** [comment]
Same here.
The render function of the page _is_ run, but something prevents Ionic to show it.

```▶ npm list | grep @ionic
├─┬ @ionic-native/core@5.17.0
├─┬ @ionic-native/hotspot@5.17.0
├─┬ @ionic/react@4.11.5
│ ├─┬ @ionic/core@4.11.5
├─┬ @ionic/react-router@4.11.5
```

**Comment by elylucas** [comment]
Hi @kodemi,

I believe this would be working now from another fix that got into 4.11.6. Could you try that out and report back here if it didn't help?

Thanks

**Comment by kodemi** [comment]
Hi @elylucas,
Yes, now it works in 4.11.6👍🏼
